src/components/Canvas/Canvas.wrapper.tsx

```diff
@@ -83,6 +83,9 @@ export class CanvasWrapper extends React.Component<ICanvasWrapperProps, IState>
     const { offsetX, offsetY } = this.state
     const { zoom } = config
 
+    if (typeof ComponentInner === 'undefined') throw new Error('ComponentInner is not defined')
+    if (typeof ComponentOuter === 'undefined') throw new Error('ComponentOuter is not defined')
```

Comment:
`null` is ook niet geldig. Dus grofweg het makkelijkst leesbaar is dit:
```suggestion
    if (!ComponentInner) throw new Error('ComponentInner is not defined')
    if (!ComponentOuter) throw new Error('ComponentOuter is not defined')
```